rtengine/curves.h

```diff
@@ -1611,6 +1611,31 @@ class WavOpacityCurveWL
     }
 };
 
+class IcmOpacityCurveWL
```

Comment:
You can attribute this to my incompetence in C++, or an oversight, or inattention. In short, I will try to correct it. At the same time, it will allow me to test my environment (Msys, etc.), my access to Github, etc.
I will push a first commit with 2 changes - to see. Each compilation takes between 12 and 25 minutes.

I hope all will work :)